pkg/operator/validations.go

```diff
@@ -39,22 +39,10 @@ func ValidateDurationField(durationField string) error {
 
 // CompareScrapeTimeoutToScrapeInterval validates value of scrapeTimeout based on scrapeInterval
 func CompareScrapeTimeoutToScrapeInterval(scrapeTimeout, scrapeInterval monitoringv1.Duration) error {
```

Comment:
not sure why we put this function in the operator package, it might be good to move it to the prometheus package and merge it with validateConfigInputs()?